engine/src/main/java/org/destinationsol/game/ship/ShipBuilder.java

```diff
@@ -29,16 +29,13 @@
 import com.badlogic.gdx.physics.box2d.joints.PrismaticJoint;
 import com.badlogic.gdx.physics.box2d.joints.PrismaticJointDef;
 import org.destinationsol.assets.json.Validator;
+import org.destinationsol.game.*;
```

Comment:
Please forbid star imports inside IDEA, then fix this. https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports